plugins/ufm_consumer_plugin/scripts/start_ufm.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# configure consumer UFM
+./config_consumer.sh
+status=$?
+if [ $status -ne 0 ]; then
+    echo "Failed to config UFM. UFM consumer plugin will not start."
```

Comment:
the reason for the failure will be printed from the called script - right?

Reply:
yes